### DISCUSSION TITLE
Community building functionality

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#398] [Ideas] [answered]
We are currently onboarding innovation-communities on the platform. I have a few suggestions for future functionality that would really help us:
- Event calendar: for meetups and other relevant events
- Repository: where we can share relevant documents, etc (we don't necessarily need to upload documents on the hub, but have the ability to create links to documents etc in a designated repository-field). 
- Visibility of challenge/opportunity lead and contact button: a quick way to contact a challenge or opportunity lead
- Latest news or "what's new" carousel: I think it would be great if we can pin community updates (on challenge level) and show them in for example a carousel. Then we can really show off what is happening within a community.

**Comment by techsmyth** [comment, marked as answer]
Hi Flori,

Captured this in a placeholder epic: https://github.com/alkem-io/alkemio/issues/408

Quite a few items to unpick above, which would then translate into multiple separate epics. Suggest we plan a roadmap session for first week of Sept as by then we should be done with the bulk of the already prioritized items. 

If there is any one of the above that should be prioritized highly near term please let me know and we can already split that one out. 

Cheers, Neil

**Reply by FloriSpoelstra** [reply]
Hi Neil, thanks and good plan to have a roadmap session in sept. I will check with some of our communities what they would like to see prioritized.

**Reply by techsmyth** [reply]
Super, can you suggest a couple of windows of say 2 hours? Suggest we keep it to a small enough group for a first pass - so yourself and myself and maybe Rene?
